src/widgets/mentors-docs/ui/additional/ui/additional.tsx

```diff
@@ -0,0 +1,23 @@
+import classNames from 'classnames/bind';
+import { DiscordIcon, TelegramIcon } from '@/shared/icons';
+import { SocialMediaItem } from '@/shared/ui/social-media-item';
+
+import styles from './additional.module.scss';
+
+const cx = classNames.bind(styles);
+
+type AdditionalProps = {
+  text: string;
+  linkTelegram: string;
+  linkDiscord: string;
+};
+
+export const Additional = ({ text, linkDiscord, linkTelegram }: AdditionalProps) => {
+  return (
+    <div className={cx('additional')}>
+      {text}
+      <SocialMediaItem title="Telegram" href={linkTelegram} icon={TelegramIcon()} />
```

Comment:
I suppose it will be better to make it text-links or move icons below

Reply:
I'm thinking of doing something like
![image](https://github.com/user-attachments/assets/c6745752-f89f-46a5-bee9-4b1be1f6f586)

And at the next meeting, we can discuss

Reply:
Moved icons below text